### ISSUE TITLE
Baseline Length of Stay (LoS) demo on MIMIC

### ISSUE THREAD

**Issue by LaRiffle** [OpenMined/PySyft#3768]
## Description

We want to reproduce the setting of the following paper: https://www.sciencedirect.com/science/article/pii/S1532046418300716

Where is showcased the training of a model to predict the LoS on the healthcare MIMIC dataset

## Breakdown

- Get a preprocessed version of MIMIC #3765
- Create the model #3767
- Create the training script according to the paper #3766

The demo produced should have an accuracy matching the one from the paper.
